### DISCUSSION TITLE
Блокнутые по IP сайты

### DISCUSSION THREAD

**Discussion by Ptalemon** [bol-van/zapret#1004] [Q&A]
Появилось желание разобраться в случае чего, как добавить к обходимым Zapretом сайтам те, которые авторы не ввели (к примеру тот же недавно упомянутый e621.net).
Мало чего смыслю в работе сетей, так что решил обратиться именно с таким вопросом(

**Comment by bol-van** [comment]
zapret не может обойти блокировки по ip
их нужно проксировать или VPNить

**Comment by numbereleven-a** [comment]
или искать альтернативные ip [в теме есть варианты](https://ntc.party/t/%D0%B4%D0%BE%D0%BC%D0%B5%D0%BD%D1%8B-instagram-%D0%B8-facebook/11822)

**Reply by Ptalemon** [reply]
Если нашёл - то как дальше? Просто закинуть в список айпи где-то в Zapret?

**Reply by numbereleven-a** [reply]
hosts

**Reply by Ptalemon** [reply]
Поиском не находит. Хотя вроде качал с этого гита

**Reply by numbereleven-a** [reply]
> Поиском не находит. Хотя вроде качал с этого гита

![7d11d528babd8b61aff5aca9201c0d19](https://github.com/user-attachments/assets/9e6f26dc-17f7-44f5-8850-7b369e7c4035)
